timeclock.py

```diff
@@ -82,17 +83,32 @@ def setup_roles():
     db.session.add(administrator)
     db.session.commit()
 
-
 @app.cli.command("create_dev_users")
 def create_dev_users():
     """Create users for development."""
-    # Administrator
     faker = Faker()
+
+    # Hardcoded administrator for local dev convenience
+    admin = User(
```

Comment:
The other admin that's created has a random email address, so you need to check the logs, and it's not validated, so you need to go through a password reset process, using a password that matches the requirements (like having special characters) - these requirements make it take significantly longer to get into the admin UI